components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/OAuth2ServiceTest.java

```diff
@@ -172,4 +213,174 @@ public void testIdentityOAuth2Exception() throws Exception {
         assertFalse(oAuth2ClientValidationResponseDTO.isValidClient());
     }
 
+    @Test
+    public void testIssueAccessToken() throws IdentityException {
+
+        OAuth2AccessTokenRespDTO tokenRespDTO = new OAuth2AccessTokenRespDTO();
+        AccessTokenIssuer accessTokenIssuer = mock(AccessTokenIssuer.class);
+        mockStatic(AccessTokenIssuer.class);
+        when(AccessTokenIssuer.getInstance()).thenReturn(accessTokenIssuer);
+        when(accessTokenIssuer.issue(any(OAuth2AccessTokenReqDTO.class))).thenReturn(tokenRespDTO);
+        assertNotNull(oAuth2Service.issueAccessToken(new OAuth2AccessTokenReqDTO()));
+
+        when(accessTokenIssuer.issue(any(OAuth2AccessTokenReqDTO.class)))
```

Comment:
don't check normal flow and exception in one test. Create another test for exception